tests/integration/backward_compatible/proxy/vector_collection_test.py

```diff
@@ -181,7 +181,21 @@ def test_backupCount_valid_values_pass(self):
             name, [IndexConfig("vector", Metric.COSINE, 3)], backup_count=2, async_backup_count=2
         )
 
-    def test_backupCount(self):
+    def test_backupCount_max_value_pass(self):
```

Comment:
Would it be possible to use lower snake case exclusively for function names, like `test_backup_count_max_value_pass`?
That's the expected Python style.
I've missed that in previous that tests that involve the backup count.
I'll send a PR to fix them.